### ISSUE TITLE
не выкладывать архивы

### ISSUE THREAD

**Issue by kmike** [dialogue-evaluation/morphoRuEval-2017#10]
Уже, наверное, поздно, но лучше не запаковывать данные в .zip файлы, а просто коммитить их как есть. 

git файлы и так жмет, т.е. история займет столько же места, если файлы не жать - а скорее даже меньше, т.к. при изменении файла нужно будет только дифф сохранить, а он гораздо меньше для текстовых данных.

**Comment by nvanva** [comment]
Да, тоже хотел попросить выкладывать незаархивированные материалы - особенно оценочный скрипт, так можно легко увидеть, что изменилось в файлах в новых коммитах. Думаю, разархивировать никода не поздно%)
